client/src/components/Header/Header.js

```diff
@@ -267,7 +267,7 @@ export const Header = ({ isDark = false, loaded }) => {
             <nav className="navbar site-navbar offcanvas-active navbar-expand-lg navbar-light">
               {/* <!-- Brand Logo--> */}
               <div className="brand-logo">
-                {isMobile ? (
+                {narrowScreen ? (
```

Comment:
Perhaps not a part of this PR, but I'd say just remove that `narrowScreen`stuff. Tailwind already supports this. So we could just show/hide those element that uses this variable with css